chromadb/cli/env.py

```diff
@@ -0,0 +1,100 @@
+import json
+import os
+import time
+import traceback
+
+import typer
+import chromadb
+from chromadb.api.types import SystemInfoFlags
+from chromadb.utils import system_info_utils
+
+env_app = typer.Typer()
+
+try:
+    from rich import print as rprint
+except ImportError:
+    rprint = typer.echo
+
+
+@env_app.command(help="Local and remote Chroma system information")  # type: ignore
+def info(
```

Comment:
IS there a good reason for nesting this under env? Why not just make .env() return the env?

Reply:
The reason is that there are two sub-commands bundled under `env`:

- Info - to printout local/remote info
- rstat - for monitoring CPU/mem utilization which is a continuous streaming of that info from the server

Reply:
I don't see `rstat`?